### PR TITLE
Maptastic

### PR DESCRIPTION
**Pull request by mikebelanger**
New package:

**Extern:** Written by hand.

### CODE REVIEW

File: maptastic/build.boot [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+       :description "Javascript/CSS projection mapping utility."
+       :url         "https://github.com/glowbox/maptasticjs"
+       :license     {"Copyright (C) 2015 by Glowbox LLC"
+                     "https://raw.githubusercontent.com/glowbox/maptasticjs/master/license.txt"}
+       :scm         {:url "https://github.com/cljsjs/packages"}})
+
+(deftask package []
+  (comp
+   (download :url
+             "https://raw.githubusercontent.com/glowbox/maptasticjs/master/build/maptastic.min.js"
```
Comment (Deraen) [inline maptastic/build.boot]: Please use specific version or git sha here (if library is not versioned), for reproducible build.

Comment (Deraen) [inline maptastic/build.boot]: And like with versions, please save the sha in a def, like +lib-version+.

Comment (Deraen) [inline maptastic/build.boot]: Like in this package: https://github.com/cljsjs/packages/blob/a486e6af0389a1d2754e3a21e7772f082b208815/emojione-picker/build.boot#L12

Comment (mikebelanger) [inline maptastic/build.boot]: Sure - to be sure you mean the sha of the original repo?

**Comment by Deraen** [comment]
I merged this with some fixes:

- It is important that `+lib-version+` and `+version+` are defined (the names must be these)
- No need to require built-in tasks namespace
- `:version` must use `+version+`, `+version+` must contain the build identifier `-0`
- Fixed indents in package task
- Removed `show` task which did nothing

**Comment by mikebelanger** [comment]
Oops - thanks again!

**Pull request merged**